conf/application.conf

```diff
@@ -130,6 +130,10 @@ google-analytics {
   host = auto
 }
 
+tracking-consent-frontend {
+  gtm.container = "a"
```

Comment:
are you creating app-config PRs for adding actual tracking code ?  or is the tracking code not ready yet ?

Reply:
it's just a container a,b,c,d,e or f and ticket says to use a, not sure how it actually works behind the scenes